.github/workflows/publish_operator_image_and_bundle.yaml

```diff
@@ -303,7 +307,25 @@ jobs:
         git merge upstream/main
         git push --set-upstream origin main
         git checkout -b ${{ github.event.repository.name }}-${{ needs.publish-image.outputs.latest_tag}}
-        mkdir operators/${{ github.event.repository.name }}/${{ needs.publish-image.outputs.latest_tag}}
+
+        if stat "operators/${{ github.event.repository.name }}/ci.yaml" >/dev/null 2>&1; then
+          echo "File exists: $CI_YAML_PATH"
+        else
+          mkdir -p operators/${{ github.event.repository.name }}
```

Comment:
you could remove this line if you bring this `mkdir -p operators/${{ github.event.repository.name }}/${{ needs.publish-image.outputs.latest_tag}}` above